root/static/styles/entity-icons.less

```diff
@@ -8,7 +8,13 @@
 }
 
 .blankheader      { background-image: data-uri('../images/entity/blank.svg') }
-.artistheader     { background-image: data-uri('../images/entity/artist.svg') }
+.artistheader     {  background-image: data-uri('../images/entity/artist.svg');
+                    &.choir-icon      { background-image: data-uri('../images/entity/choir.svg')  }
```

Comment:
This line and all the other -icon lines except character-icon have an extra space before the closing `  }` (should be 1 space).